hypothesis-python/docs/stateful.rst

```diff
@@ -169,6 +169,57 @@ fewer examples with larger programs you could change the settings to:
 Which doubles the number of steps each program runs and halves the number of
 test cases that will be run.
 
+-----
+Rules
+-----
+
+As said earlier, rules are the most common feature used in RuleBasedStateMachine.
+They are defined by applying the :func:`~hypothesis.stateful.rule` decorator
+on a function.
+Note that RuleBasedStateMachine must have at least one rule defined and that
+a single function cannot be used to define multiple rules (this to avoid having
+multiple rules doing the same things).
+
+.. autofunction:: hypothesis.stateful.rule
+
+-----------
+Initializes
+-----------
+
+Initializes are a special case of rules that are guaranteed to be run at most
+once at the beginning of a run.
+Note if multiple initialize rules are defined, their call order is randomized
```

Comment:
I'd prefer to say something like "they may be called in any order, and that order will vary from run to run" ("randomized" isn't really a good description of what Hypothesis does)